src/main/java/seedu/address/logic/parser/PersonAttributeCommandParser.java

```diff
@@ -1,13 +1,17 @@
 package seedu.address.logic.parser;
```

Comment:
Could this be renamed to AddAttributeParser, and the common logic extracted to ParserUtil?

Reply:
yep